liquibase-core/src/test/groovy/liquibase/serializer/core/string/StringChangeLogSerializerTest.groovy

```diff
@@ -309,6 +310,8 @@ public class StringChangeLogSerializerTest extends Specification {
                 //nothing
             } else if (field.getType().equals(LoadDataChange.LOAD_DATA_TYPE.class)) {
                 //nothing
+            } else if (field.getType().equals(ColumnOwnerType.class)) {
+                //nothing
```

Comment:
Is this the correct way to handle this new enum field (that is ignored in serialization by overriding `getSerializableFields()`) @nvoxland @filipelautert? The test `general field serialization check` fails without it.

Reply:
I think it's fine, otherwise you would need to add it to most of the tests above it.

Reply:
It maybe seems best to actually set a value there to be able to test against instead of just ignoring it. Otherwise we're not really testing how it serializes a field of that type. You can maybe just randomly pick one like "table" for the test? 

But I'm also not sure what the test failure was. Was it failing because it was catching a real problem in the code and ignoring it just avoids the real bug? Or was it just "don't know what to use for a value?" and so giving value instead of ignoring it is fine? Or something else?

Reply:
Updated to set the value in the test case. It was failing because it did not know what to do with the value.